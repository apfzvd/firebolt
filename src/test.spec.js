import { shallow } from 'enzyme'
import App from './app'
import React from 'react'
const wrapper = shallow(<App />)

it('prim', () => {
  expect(wrapper.find('Button')).toHaveLength(1)
})
