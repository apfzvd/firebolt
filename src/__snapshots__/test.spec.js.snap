// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prim 1`] = `
"<Fragment>
  <Button onClick={[Function: onClick]} iconLeft=\\"whatshot\\" disabled={false} color=\\"primary\\" size=\\"medium\\" type=\\"button\\" className={[undefined]} active={false}>
    hello
  </Button>
</Fragment>"
`;
